test/test_helper.rb

```diff
@@ -16,9 +16,10 @@
   config.cassette_library_dir = 'test/vcr_cassettes'
   config.hook_into :webmock
   config.allow_http_connections_when_no_cassette = false
-  config.filter_sensitive_data('http://FAKE_TIMDEX_HOST/api/v2/') { ENV.fetch('TIMDEX_BASE').to_s }
+  config.filter_sensitive_data('http://FAKE_TIMDEX_HOST/graphql/') { ENV.fetch('TIMDEX_BASE').to_s }
```

Comment:
Can you double check these changes make sense? It seems like `TIMDEX_GRAPHQL` and `TIMDEX_BASE` are being used backward here... but if it works... 🤷🏻

Reply:
I'm pretty sure this is working for the scrubbing:

```
  config.filter_sensitive_data('FAKE_TIMDEX_HOST') { ENV.fetch('TIMDEX_HOST').to_s }
  config.filter_sensitive_data('http://FAKE_ORIGIN') { ENV.fetch('TIMDEX_UI_ORIGIN').to_s }
  config.filter_sensitive_data('http://FAKE_TIMDEX_HOST/graphql') { ENV.fetch('TIMDEX_GRAPHQL').to_s }
```

Which we'd then remove TIMDEX_BASE from .env.test

Reply:
Yeah, I was having a hard time wrapping my head around how we could trim down ENV with this change - it felt like we definitely could, but was struggling with some details.

Reply:
Let me take a stab at it now, but if it feels like it's going to drag we can punt to a future ticket yeah